docs/usage.rst

```diff
@@ -60,3 +60,23 @@ file on the host system: ::
         /data /out/out \
         participant \
         --ignore fieldmaps
+
+
+Usage tracking with Google Analytics
+------------------------------------
+
+To be able to assess usage of the software, we are recording each use of the
+CLI as an event in Google Analytics, using `popylar <https://popylar.github.io>`_
```

Comment:
Expand acronym on the first occurrence, and since the link is unlikely to be used again:
```suggestion
:abbr:`CLI (command-line interface)` as an event in Google Analytics,
using `popylar <https://popylar.github.io>`__.
``